decidim-elections/app/views/decidim/votings/polling_officer_zone/polling_officers/_polling_station.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="card card--secondary">
+  <div class="card__content">
+    <div class="card__header">
+      <h5 class="card__title">
+        <%= link_to resource_locator(election).path do %>
+          <%= content_tag :span,
+                          present(election).title,
+                          title: t("polling_station.election", scope: "decidim.votings.polling_officer_zone.polling_officers.index") %>
+          <%= content_tag :small,
+                          present(polling_officer.voting).title,
+                          title: t("polling_station.voting", scope: "decidim.votings.polling_officer_zone.polling_officers.index") %>
+        <% end %>
+      </h5>
+    </div>
+
+    <div>
+      <%= content_tag :span,
+                      t(polling_officer.role, scope: "decidim.votings.polling_officers.roles"),
```

Comment:
Are quotation marks missing here?
```suggestion
                      t("polling_officer.role", scope: "decidim.votings.polling_officers.roles"),
```

Reply:
This is OK without quotation marks, as `polling_officer.role` returns "Manager" or "President".